soh/soh/Enhancements/custom-message/CustomMessageTypes.h

```diff
@@ -38,6 +38,7 @@ typedef enum {
     TEXT_CARPET_SALESMAN_1 = 0x6077,
     TEXT_CARPET_SALESMAN_2 = 0x6078,
     TEXT_MARKET_GUARD_NIGHT = 0x7003,
+    TEXT_FISHERMAN_LEAVE = 0x409E,
```

Comment:
Is this ID the same for both child and adult fishing sessions? I'm in the process of doing all text IDs, but I won't get there for a while.

Reply:
Yup, it is!

Reply:
Cool, you can resolve this.